### PR TITLE
feat: support multiple target on useClickAway

### PR DESCRIPTION
**Pull request by Fadingvision**

### CODE REVIEW

**Review by brickspert — COMMENTED** [review]
LGTM，can you add demo and doc for this feature? thank you very much.

**Comment by Fadingvision** [comment]
@brickspert Updated

File: packages/hooks/src/useClickAway/demo/demo3.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/**
+ * title: Custom multiple DOM target
+ * desc: Support pass multiple DOM elements, or functions which returns the DOM element.
+ *
+ * title.zh-CN: 支持多个DOM对象
```
Comment (brickspert) [inline packages/hooks/src/useClickAway/demo/demo3.tsx]: 中英文之间加空格

File: packages/hooks/src/useClickAway/demo/demo3.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * desc.zh-CN: 支持直接传入多个目标对象的方式引入。
```
Comment (brickspert) [inline packages/hooks/src/useClickAway/demo/demo3.tsx]:
```suggestion
 * desc.zh-CN: 支持传入多个目标对象。
```

File: packages/hooks/src/useClickAway/index.zh-CN.md [hunk 3]
```diff
@@ -34,4 +37,4 @@ useClickAway(
 | 参数    | 说明                                         | 类型                   | 默认值 |
 |---------|----------------------------------------------|------------------------|--------|
 | onClickAway | 触发事件的函数  | (event) => void | -      |
-| target | DOM 节点或者 Ref 对象  | (() => HTMLElement) \| HTMLElement \| React.MutableRefObject | - |
+| target | DOM 节点或者 Ref 对象或者 包含该对象的数组 | (() => HTMLElement) \| HTMLElement \| React.MutableRefObject \| ((() => HTMLElement) \| HTMLElement \| React.MutableRefObject)[] | - |
```
Comment (brickspert) [inline packages/hooks/src/useClickAway/index.zh-CN.md]: `DOM 节点或者 Ref 对象，支持数组`

**Comment by Fadingvision** [comment]
@brickspert 更新了

**Review by brickspert — APPROVED** [review]
LGTM

**Review by awmleer — APPROVED** [review]
LGTM

**Pull request merged**